test/cri-containerd/policy_test.go

```diff
@@ -64,18 +64,20 @@ func policyFromOpts(t *testing.T, policyType string, opts ...securitypolicy.Poli
 		config.AllowRuntimeLogging,
 		config.AllowEnvironmentVariableDropping,
 		config.AllowUnencryptedScratch,
+		config.AllowCapabilityDropping,
 	)
 	if err != nil {
 		t.Fatal(err)
 	}
 	return base64.StdEncoding.EncodeToString([]byte(policyString))
 }
 
-func alpineSecurityPolicy(t *testing.T, policyType string, allowEnvironmentVariableDropping bool, opts ...securitypolicy.ContainerConfigOpt) string {
+func alpineSecurityPolicy(t *testing.T, policyType string, allowEnvironmentVariableDropping bool, allowCapabilityDropping bool, opts ...securitypolicy.ContainerConfigOpt) string {
```

Comment:
looks like `allowCapabilityDropping` is always `false`, maybe we don't need to update the signature for now and just always set it to `false`, unless some tests are being added that needs this. In that case though, maybe we should just change the parameters to be slices, rather than variadic parameter list.

Reply:
It isn't in Matt's CRI tests. He built off this branch I think, I'll leave this conversation to you and @matajoh

Reply:
Yes, the tests are in the seccomp branch